himl/config_generator.py

```diff
@@ -26,7 +26,7 @@
 class ConfigProcessor(object):
 
     def process(self, cwd=None, path=None, filters=(), exclude_keys=(), enclosing_key=None, remove_enclosing_key=None, output_format="yaml",
-                print_data=False, output_file=None, skip_interpolations=False, skip_interpolation_validation=False, skip_secrets=False):
+                print_data=False, output_file=None, skip_interpolations=False, skip_interpolation_validation=False, skip_secrets=False, multi_line_string=False):
```

Comment:
I think the default for `multi_line_string` can be set as `True`.

Reply:
Since this will override the global BaseRepresenter for a client importing `himl` as a library, I've set it to default `False`, let me know wdyt and I can change it.